samples/python/requirements.txt

```diff
@@ -1,2 +1,5 @@
 opencv-python==4.5.*
 numpy>=1.16.6,<=1.23.1
+# For bert_becnmark
+transformers[onnx]
```

Comment:
I also don't think it is a good idea to have one requirements.txt for all the samples. For example, "bert_benchmark.py" depends on transformers and torch which are huge dependency while others can depend on numpy only. Since anyway you have separate README for each sample, it would be better to split the requirements as well.

Reply:
Split